frontend/packages/topology/src/components/list-view/TopologyListViewUnassignedGroup.tsx

```diff
@@ -53,7 +53,7 @@ const TopologyListViewUnassignedGroup: React.FC<TopologyListViewUnassignedGroupP
   cells.push(
     <DataListCell
       key="label"
-      className="odc-topology-list-view__unassigned-label"
+      className="odc-topology-list-view__unassigned-label odc-topology-list-view__nofocus-label"
```

Comment:
Don't find any css definition for `odc-topology-list-view__nofocus-label` ?! Can we drop this again?

```suggestion
      className="odc-topology-list-view__unassigned-label"
```

Reply:
updated